core/src/components/button/button.ionic.scss

```diff
@@ -39,6 +39,11 @@
 // Button Fills
 // -------------------------------------------------------------------------------
 
+// Default Clear Color
+:host(.button-clear) {
+  --color: #{ion-color(primary, foreground)};
```

Comment:
Why move the `.ion-color` selectors that are shared by all themes?

Reply:
Same as above